src/commands/support/hud.ts

```diff
@@ -0,0 +1,14 @@
+import { CommandDefinition } from '../../lib/command';
+import { makeEmbed } from '../../lib/embed';
+import { CommandCategory } from '../../constants';
+
+export const hud: CommandDefinition = {
+    name: ['hud'],
+    description: 'How to disable in-game HUD',
+    category: CommandCategory.SUPPORT,
+    executor: (msg) => msg.channel.send(makeEmbed({
+        title: 'FlyByWire Support | Disable in game HUD',
+        description: 'Please turn off INSTRUMENTS HEADS UP DISPLAY - CHASE CAM and COCKPIT CAM in Settings > Assistance Options > User Experience ',
```

Comment:
```suggestion
        description: 'Please turn off \'INSTRUMENTS HEADS UP DISPLAY - CHASE CAM\' and \'COCKPIT CAM\' in Settings > Assistance Options > User Experience ',
```
might be better to have these in quotes